includes/class-wc-stripe-order-helper.php

```diff
@@ -440,6 +475,202 @@ public function update_stripe_setup_intent( ?WC_Order $order = null, string $int
 		$order->update_meta_data( self::META_STRIPE_SETUP_INTENT, $intent_id );
 	}
 
+	/**
+	 * Gets the Stripe customer for order.
+	 *
+	 * @since 10.0.0
+	 *
+	 * @param WC_Order|null $order
+	 * @return false|string|null
+	 */
+	public function get_stripe_customer( ?WC_Order $order = null ) {
+		if ( is_null( $order ) ) {
```

Comment:
Given the type for the argument, I think we can simplify these null checks to use a direct equality check rather than calling `is_null()`.
```suggestion
		if ( null === $order ) {
```

Reply:
Updated all checks in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4705/commits/10125f0c244606e5201c5c9546a8a8c280e8cd70